code/SiteWideContentReport/Model/SitewideContentTaxonomy.php

```diff
@@ -0,0 +1,76 @@
+<?php
+
+namespace SilverStripe\Reports\SiteWideContentReport\Model;
```

Comment:
```suggestion
namespace SilverStripe\Reports\SiteWideContentReport\Extensions;
```
And rename the `Model` dir to `Extensions`

Reply:
Done